SPECS/grub.spec

```diff
@@ -225,6 +228,10 @@ do
 done
 install -m 755 %{grubefiname} $RPM_BUILD_ROOT/boot/efi/EFI/%{efidir}/%{grubefiname}
 install -m 755 %{grubeficdname} $RPM_BUILD_ROOT/boot/efi/EFI/%{efidir}/%{grubeficdname}
+# XCP-ng: Add fallback for when all boot entries fail
+# (buggy UEFI implementation, NVRAM error, user error in configuring boot entries, etc... could all cause this)
```

Comment:
Add also a comment saying that the fallback being a copy of grubx64.efi, it will still look for its configuration at EFI/xenserver/grub.cfg